### PR TITLE
PCHR-2144: Temporary _temp_custom.scss style for document modal in SSP

### PR DESCRIPTION
**Pull request by bmnepali**
## Overview
As the document modal is injected for T&A, but styles are are not. So the document modal window gets broken in SSP. So, adding custom temporary styles for the modal in SSP to look like in T&A dashboard. 

## Before
Modal after modal is injected form T&A
screen shot 2017-05-25 at 6 17 22 pm

## After
Modal after adding custom styles 
screen shot 2017-05-25 at 6 04 38 pm

Modal displayed in Documents section in SSP for staff
screen shot 2017-06-01 at 1 19 20 pm

More refined Modal (style wise)
screen shot 2017-06-05 at 1 35 55 pm
screen shot 2017-06-05 at 1 36 20 pm

### CODE REVIEW

File: civihr_default_theme/scss/civihr_default_theme.style.scss [hunk 1]
```diff
@@ -95,13 +95,11 @@
 // Custom CiviHR Theme CSS
 // --------------------------------------------------
 @import "base/custom";
-
+@import "base/_temp_custom";
```
Comment (AkA84) [inline civihr_default_theme/scss/civihr_default_theme.style.scss]: put this at the end of the file in a "Temporary" section

Comment (AkA84) [inline civihr_default_theme/scss/civihr_default_theme.style.scss]: also can you give it a more meaningful name? Like `_task-assignments-modal-temp.scss`

Comment (AkA84) [inline civihr_default_theme/scss/civihr_default_theme.style.scss]: also, when you `@import` a partial, you don't need to specify the `_` (see other imports in the same file)

**Review by AkA84 — CHANGES_REQUESTED** [review]

File: civihr_default_theme/scss/civihr_default_theme.style.scss (hunk 1 above)
Comment (bmnepali) [reply civihr_default_theme/scss/civihr_default_theme.style.scss]: right! will fix it

Comment (bmnepali) [reply civihr_default_theme/scss/civihr_default_theme.style.scss]: will do

File: civihr_default_theme/scss/base/_task-assignments-modal-temp.scss [hunk 2]
```diff
@@ -0,0 +1,248 @@
+// This is the temporary styles to male the new Document modal look similar in SSP
```
Comment (dhavalsharma) [inline civihr_default_theme/scss/base/_task-assignments-modal-temp.scss]: is male make?

**Review by dhavalsharma — CHANGES_REQUESTED** [review]

**Comment by dhavalsharma** [comment]
can you also specify the source of this style file for reference here ?

File: civihr_default_theme/scss/base/_task-assignments-modal-temp.scss (hunk 2 above)
Comment (bmnepali) [reply civihr_default_theme/scss/base/_task-assignments-modal-temp.scss]: Exactly! will fix it

**Comment by bmnepali** [comment]
@dhavalsharma Most of the styles re custom. Some of the styles taken from 
1. uk.co.compucorp.civicrm.tasksassignments/scss/civihr/partials/_datepicker.scss

**Review by dhavalsharma — APPROVED** [review]

**Review by AkA84 — APPROVED** [review]

**Review by AkA84 — APPROVED** [review]

**Review by AkA84 — APPROVED** [review]

**Review by AkA84 — APPROVED** [review]

**Pull request merged**